packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/serverless-function-execution-result.dto.ts

```diff
@@ -1,13 +1,32 @@
 import { Field, ObjectType } from '@nestjs/graphql';
 
-import { IsObject } from 'class-validator';
+import { IsObject, IsOptional } from 'class-validator';
 import graphqlTypeJson from 'graphql-type-json';
 
 @ObjectType('ServerlessFunctionExecutionResult')
 export class ServerlessFunctionExecutionResultDto {
   @IsObject()
   @Field(() => graphqlTypeJson, {
     description: 'Execution result in JSON format',
+    nullable: true,
   })
-  result: JSON;
+  data?: JSON;
+
+  @Field({ description: 'Execution duration in milliseconds' })
+  duration: number;
+
+  @Field({ description: 'Execution status code' })
+  status: 200 | 500;
```

Comment:
not a big fan of having a http status there. I feel we should use an enum for that.

Reply:
ok